src/plugins/process_anchors.js

```diff
@@ -17,7 +17,8 @@ import path from 'path'
 function isInternalDomain(url) {
   try {
     const domain = new URL(url)
-    return domain.hostname.endsWith('gsa.gov') || domain.protocol=='mailto:'
```

Comment:
Just want to double check  — this will cause links to training.smartpay.gsa.gov (it's linked in a few spots in the FAQs) and the 889 tool (linked on the front page) to get the external link icon while links to the main gsa.gov site don't. Is that the intent?

Reply:
Yes, this is the intent. The external site icon should display for all links that will navigate from the current site. Links navigating within the current site should not have the external link.

Reply:
@felder101 if you go this route, you probably need to check links like https://federalist-ab31a10d-375d-4040-9324-1ae94e8a36b9.sites.pages.cloud.gov/preview/gsa/smartpay-website/GSA/ProductionRelease/policies-and-audits/smart-bulletins/010/ and https://federalist-ab31a10d-375d-4040-9324-1ae94e8a36b9.sites.pages.cloud.gov/preview/gsa/smartpay-website/GSA/ProductionRelease/policies-and-audits/smart-bulletins/020/ that directly to smartpay.gsa.gov because now this gets an external link icon even through it's to the same site.

Reply:
Thanks for the review. We will analyze the links within the site to make sure we are consistent throughout within our current sprint. Looks like there might be some inconsistencies with the external link icon.